src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule72thesummaryfragment/InputIncorrectSummaryJavaDocCheck.java

```diff
@@ -85,32 +85,32 @@ void foo6() {}
       */
     InnerInputCorrectJavaDocParagraphCheck anon = new InnerInputCorrectJavaDocParagraphCheck() {
 
-        /**
-         * Some Javadoc.
-         */
-        public static final byte NUL = 0;
+            /**
```

Comment:
@sagar-shah94 I assume you are referring to this.
It looks like you indented this code over to the right 4 spaces.
I don't see a reason why we need to make this change, so I was asking you to undo the indent and move them to the left 4 spaces.